02LessonTwo/in-class-project/index.html

```diff
@@ -1,10 +1,39 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Muffin Market</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+  <center><img src="https://s-media-cache-ak0.pinimg.com/736x/80/a1/c5/80a1c5ce85954a56487e33b5c8e8b466.jpg" alt="Muffin Market"/></center>
+  <h1 align="center">Muffin Market</h1>
+  <h2 align="center"><em>Fluffy muffin  for all!</em></h2>
+  <br>
+  <br>
+  <br>
+  <table border="1" align="center">
+    <tr>
+      <th colspan="3"><h3>Muffin for sales!</h3></th>
+      </tr>
+    </th>
+    <tbody>
+        <td><img src="https://s-media-cache-ak0.pinimg.com/736x/80/a1/c5/80a1c5ce85954a56487e33b5c8e8b466.jpg" alt="Muffin Market"/></td>
+        <td><img src="http://a66c7b.medialib.glogster.com/dianahan/media/cd/cd1fe442dd38ab207e86bfb0c7e92894424fa4cb/muffin2.png" alt="Muffin Market"/></td>
+        <td><img src="https://s-media-cache-ak0.pinimg.com/564x/0e/e3/6f/0ee36f74077f3ae6e829e1cf103d6efc.jpg" alt="Muffin Market"/></td>
+    <tbody>
+      <tr>
+        <td><p align=center>Blueberry FREE</p></td>
+        <td><p align=center>Blueberry FREE</p></td>
+        <td><p align=center>Blueberry FREE</p></td>
+      </tr>
+    </th>
+  </table>
+  </ul>
+  <br>
+  <br>
+  <br>
+<center><a href="http://google.com">Muffin Market</a></center>
+<p align="center">Phone: 555-555-5555</p>
+<center><a href="mailto:123@web.com">Contact me: YO YO</a></center>
```

Comment:
Fantastic effort! Detailed and thorough. Your code is organized like a professional!